src/main/webapp/script.js

```diff
@@ -13,31 +13,47 @@
 // limitations under the License.
 
 
-var url = "";
-var loggedIn = false;
+/** *********************************************************************
+ * Utility methods/onload methods (all pages)
+ ***********************************************************************/
+let url = '';
+let loggedIn = false;
 
-/* determines which stylesheet to use and generates nav bar*/
-function loadActions() {
+/**
+ * determines which stylesheet to use and generates nav bar
+ *
+ * @param {function} doAfter used to specify actions during checkLogin()
+ */
+function loadActions(doAfter) {
   const styleSheetElement = document.getElementById('style');
   if (window.innerWidth >= window.innerHeight) {
```

Comment:
I will update the eslint config to include pre-defined globals such as 'window'. For now, ignore this lint error.

Reply:
ok, thank you!